documentation/reference/document-select-language.html

```diff
@@ -223,6 +236,13 @@ <h4 id="comparisons-with-missing-fields-null-values">Comparisons with missing fi
 To work around this issue, "short-circuiting" using a field presence check may be used:
 <code>foo.expires_at_time and foo.expires_at_time > now()</code>.
 </p>
+<div class="alert alert-warning">
+If your selection references imported fields, <code>null</code> will be returned for any imported field when
```

Comment:
Perhaps we should add a short example on how such GC expression should be formulated (taking null into account)? Could even use the grandparent / parent / child setup described further down.

Reply:
There is an example in the paragraph above it, but it would make sense to add one explicitly